### PR TITLE
docs: add slack channel info

### PR DESCRIPTION
**Pull request by Eomm**
Hi,
I have added to readme the slack information, referring to issue #118 

Hope this help

### CODE REVIEW

**Comment by mhdawson** [comment]
@Eomm, you might wonder why I removed "official".  It's because as a partner community there is only a loose association and I want to avoid any concern over making it look like the node-js slack is more closely associated with the project than it is.

File: README.md [hunk 1]
```diff
@@ -53,10 +53,20 @@ The issue will provide schedule logistics as well as
 an agenda, links to meeting minutes, and
 information about how to join as a participant or a viewer.
 
+### Communications
+
+The working group can chat on the[Node.js slack](https://node-js.slack.com/)
```
Comment (ljharb) [inline README.md]:
```suggestion
The working group can chat on the [Node.js slack](https://node-js.slack.com/)
```

File: README.md [hunk 2]
```diff
@@ -53,10 +53,20 @@ The issue will provide schedule logistics as well as
...
+in the channel `#package-maintenance` in order to continue conversations beyond the meetings.
+All the contributors are expected to follow the [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)
+of Node.js project.
```
Comment (ljharb) [inline README.md]:
```suggestion
of the Node.js project.
```

**Comment by Eomm** [comment]
@mhdawson I understand what you mean and I agree with your consideration 👍 

@ljharb I have applied your suggestions, thank you

**Comment by mhdawson** [comment]
@ljharb if you can review/approve I'll go ahead and land.

**Pull request merged**